.golangci.yml

```diff
@@ -0,0 +1,45 @@
+linters-settings:
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - style
+      - performance
+    disabled-checks:
+      - whyNoLint
```

Comment:
Can we add `paramTypeCombine` to the disabled checks? I'm willing to adapt to many of the Go conventions, but I feel like this one sacrifices readability in favor of a slightly shorter function signature.